jsonrpc/filter_manager_test.go

```diff
@@ -511,9 +511,7 @@ func newMockWsConnWithMsgCh() (*mockWsConn, <-chan []byte) {
 func TestHeadStream(t *testing.T) {
```

Comment:
as far as we testing data races, could we have a test for it? something simple like calling each API in a separate goroutine with tickers like 1mcs?

Reply:
Let me know what you think. I added a new `TestHeaderStream_Concurrent` test.